pkg/manager/member/tidb_discovery_manager.go

```diff
@@ -249,6 +249,8 @@ func (m *realTidbDiscoveryManager) getTidbDiscoveryDeployment(obj metav1.Object)
 		})
 	}
 
+	podLabels := util.CombineStringMap(l.Labels(), baseSpec.Labels())
```

Comment:
`getDiscoveryMeta` function also needs to update?

Reply:
our other statefulset didn't include user-specified labels and annotations yet. so let this deployment as is now, if needed, we can added later.

Reply:
ok